mscore/plugin/api/elements.cpp

```diff
@@ -70,6 +70,20 @@ void Note::setTpc(int val)
             set(Pid::TPC2, val);
       }
 
+//---------------------------------------------------------
+//   Note::addInternal
+//---------------------------------------------------------
```

Comment:
Please add a `/// \since MuseScore 3.3.1` here

Reply:
Since comments now in `isChildAllowed()`, `add()`, `addInternal()` and `remove()`